compiler/IREmitter/Payload/codegen-payload.c

```diff
@@ -622,6 +622,11 @@ int sorbet_rubyArrayLen(VALUE array) {
     return RARRAY_LEN(array);
 }
 
+SORBET_INLINE
+VALUE sorbet_rubyArrayAref(VALUE array, int i) {
+    return RARRAY_AREF(array, i);
+}
+
```

Comment:
Do you want to add a note here that this differs from the C function implementing the `Array#[]` intrinsic because this basically treats `array` as a C array and directly indexes it without bounds checking?

(Or maybe: change the name of the `sorbet_` wrapper to make that more clear, like `sorbet_rubyArrayArefUnchecked`?)

Reply:
Changing the name sounds like a good idea, I'll do that.